description/description.shacl.ttl

```diff
@@ -594,6 +700,49 @@
         sh:message "schema:familyName n'est pas de type string"@fr ;
     ] .
 
+<#QuantitativeValueShape> a sh:NodeShape ; 
+    sh:targetClass schema:QuantitativeValue ;
+    [
+        a sh:PropertyShape ;
+        sh:path schema:unitCode ;
+        sh:nodeKind sh:Literal ;
+        sh:datatype xsd:string ;
+        sh:name "unit code"@en ;
+        sh:name "eenheidcode"@nl ;
+        sh:name "code d'unité"@fr ;
+        sh:severity sh:Violation ;
+        # Voor afwerken model: 'string' of 'xsd:string' schrijven in de message?
```

Comment:
maakt niet zo veel uit, is vooral voor ons. Maar we moeten hier wat regelmaat in brengen. Kan je evt hier https://meemoo.atlassian.net/wiki/spaces/MET/pages/3512631297/Modelleren+101 een pagina aanmaken met 'stijlregels' voor het maken van SHACL?